source/simulator/helper_functions.cc

```diff
@@ -848,6 +848,13 @@ namespace aspect
       double temp[2];
       Utilities::MPI::sum (my_temp, mpi_communicator, temp);
 
+      Assert (my_area > 0,
```

Comment:
what if one processor has no boundary part? 
```suggestion
      Assert (my_temp[1] > 0.0,
```

Reply:
Oh! Yes, fixed!